repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -56,6 +56,8 @@ class Interpreter(prompt0: Ref[String],
       |""".stripMargin
 
   val SheBang = "#!"
+  var lastEx:Throwable=null
```

Comment:
I think lastException is a more verbose name so it should be exposed in trait repl for users to call,lastEx is just an inside variable in interpreter trait to which lastException points